src/main/java/at/hannibal2/skyhanni/features/inventory/chocolatefactory/ChocolateFactoryAPI.kt

```diff
@@ -80,7 +79,17 @@ object ChocolateFactoryAPI {
     var shrineIndex = 41
     var coachRabbitIndex = 42
     var maxRabbits = 395
-    var maxMilestoneChocolate = 700_000_000_000L
+    var chocolateMilestones = listOf(
```

Comment:
Use a TreeSet instead of a list. Makes the search way quicker (with .higher(value))

Reply:
Also you need to set this value in the RepoUpdate event not statically.

Reply:
Sorry it is called ```RepositoryReloadEvent```

Reply:
What's the difference then when the existing one was set statically?

Reply:
Oh, I didn't see the architecture of the code. It handles the replace bellow in the RepositoryReloadEvent. So nevermind. But the TreeSet change should be done.